examples/storage-examples/gluster-examples/README.md

```diff
@@ -0,0 +1,59 @@
+# OpenShift and GlusterFS – running NGINX applications with Storage
+---
+### Environment:
+This environment consists of 4 hosts: 
+* 2 GlusterFS nodes consisting of the gluster cluster (gluster1.rhs and gluster2.rhs)
+* 2 RHEL7 Atomic Hosts running OpenShift and GlusterFS Client (master= ose1.rhs and node1 = ose2.rhs)
+
+OpenShift is installed from [OpenShift Admin Guide – quick installation](https://docs.openshift.com/enterprise/3.0/admin_guide/install/quick_install.html) and operational (meaning the server is running and you can login to the OpenShift Console via the GUI - https://master-host:8443/console).  Also the gluster cluster has been tested with all RHEL Atomic Nodes and is working properly and accessible to the OpenShift nodes.  
+
+Below is a summary of the Gluster Volume Info used for this example:
+
+
+| Attribute       | Value                 |
+|:--------------- | --------------------- | 
+| Volume Name     | myVol1                |
+| Status          | Started               |
+| Num Bricks      | 2                     |
+| Brick1          | gluster1.rhs:/mnt/brick1/gv0 |
+| Brick2          | gluster2.rhs:/mnt/brick1/gv1 |
+
+| Attribute       | Value                 |
+|:--------------- | --------------------- |
+| Volume Name     | myVol2                |
+| Status          | Started               |
+| Num Bricks      | 2                     |
+| Brick1          | gluster1.rhs:/mnt/brick1/gv1 |
+| Brick2          | gluster2.rhs:/mnt/brick1/gv2 |
+
```

Comment:
The bricks of myVol1 and myVol2 overlap on /mnt/brick1/gv1 which will not work
Also, The brick names are a slightly confusing here.  Convention is along the lines of:

/mnt/brick{1..n}/volumeName 

And in this case should be: /mnt/brick1/myVol1 and /mnt/brick1/myVol2 on both hosts.  

or if sharing a mass of space mounted at /mnt/bricks:

/mnt/bricks/brick{1..n}/volName